src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -716,6 +716,11 @@ public void onWorldTick(@NotNull TickEvent.WorldTickEvent event)
                         });
             }
 
+            if((event.world.getWorldTime() % CITIZEN_CLEANUP_TICK_INCREMENT) == 0 && !areAllColonyChunksLoaded(event))
+            {
+                Log.getLogger().info("Couldn't clean up all citizens because colonyChunks weren't loaded");
+            }
+
             //  Cleanup Buildings whose Blocks have gone AWOL
             cleanUpBuildings(event);
```

Comment:
Does this load chunks? If so we should only do this if the chunks are loaded.

Also, we should do a areAllColonyChunksLoaded check once and store the result.

Not sure when it was merged in, but I want to comment on the areAllColonyChunksLoaded method, but I can't. We should use a MutableBlockPos and just change that, so that we don't have to make a BlockPos instance at every iteration.